packages/tables/src/Columns/Concerns/CanSaveState.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Filament\Tables\Columns\Concerns;
+
+use Closure;
+use Filament\Tables\Columns\Column;
+use Filament\Tables\Contracts\HasRelationshipTable;
+use Filament\Tables\Contracts\HasTable;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Support\Str;
+
+trait CanSaveState
+{
+    protected ?Closure $saveStateUsing = null;
+
+    public function bootCanSaveState()
```

Comment:
Where is this being called from?

Reply:
Oh damn. I am getting to comfortable with Livewire booting these traits 🙈

Reply:
What do you think is the best way? Renaming it to `setUp()` and importing it to `Column::setUp()` (if instance of Editable)?

Reply:
Just call it from `setUp()`, I don't think it needs to be in the trait. Please test it too

Reply:
Hm, I'd like to keep it in the trait and keep the `setUp()` method clean. But we could also move it to `Column::setUp()` if you prefer.